tests/ttnn/unit_tests/operations/test_repeat.py

```diff
@@ -23,3 +23,21 @@ def test_repeat(device):
     output = ttnn.to_torch(output)
 
     assert_with_pcc(torch_result, output, 0.9999)
+
+
+def test_repeat_with_tile_padding(device):
```

Comment:
Could you please add more tests? Just want to make sure that all possible combinations of inputs works.
Will be extremely helpful! Thanks in advance!

Reply:
We'll be adding a bunch of tests for pad as part of sweep/generality efforts. This PR is just intended to get the models team unblocked on a particular case, but I could generalize the test slightly for now.

Reply:
I added a basic repeat sweep.